rewards/snapshot/chain_snapshot.py

```diff
@@ -41,6 +41,33 @@ def chain_snapshot(chain: Network, block: int) -> Dict[str, Snapshot]:
     return balances_by_sett
 
 
+def total_harvest_sett_snapshot(
+        chain: Network, start_block: int, end_block: int,
+        sett: str, blacklist: bool, number_of_snapshots: int,
+) -> Snapshot:
+    """
+    Get a snapshot for total harvest period. That should sum up all Snapshots balances for
+    the number_of_snapshots
+    """
+    assert end_block >= start_block
+    snapshot = sett_snapshot(chain, start_block, sett, blacklist)
+    if end_block == start_block:
+        return snapshot
+    snapshot += sett_snapshot(chain, end_block, sett, blacklist)
+    rate = int((end_block - start_block) / number_of_snapshots)
+    # If rate == 0 it means that number of snapshots is too big, and it cannot be calculated
+    # properly
+    # For ex: start block 13710328, end block 13710338 and num of snaps == 14
```

Comment:
Great sanity check!  I'd simplify the example so people can use mental math to see the exact correlation.

`For ex: start block = 100, end block = 110 and num of snaps = 14`

Reply:
Done